Configuration/DataProcessing/python/Utils.py

```diff
@@ -133,9 +133,45 @@ def dqmSeq(args,default):
         return ':'+('+'.join(args['dqmSeq']))
     else:
         return default
-            
+
 def gtNameAndConnect(globalTag, args):
     if 'globalTagConnect' in args and args['globalTagConnect'] != '':
         return globalTag + ','+args['globalTagConnect']        
     # we override here the default in the release which uses the FrontierProd servlet not suited for Tier0 activity
     return globalTag +',frontier://PromptProd/CMS_CONDITIONS'
+
+def customise_HPbeamspot(process):
+
+    # write to sqlite the HP tag and use the HP medatata for uploading it  to the dropbox
+    # ByLumi
```

Comment:
IMO, this method does not belong in this file (Configuration/DataProcessing/python/Utils.py).
Based on implementation here, perhaps  Calibration/TkAlCaRecoProducers/python is a good place.